Sloth.Web/Views/Transactions/Index.cshtml

```diff
@@ -5,9 +5,21 @@
 }
 
     <div class="container">
-        <div class="alert alert-warning" role="alert">
-        There are N pending transaction, <a href="#">click here</a> to view them
-        </div>
+    @if (Model.PendingApprovalCount > 0)
+    {
+        @if (Context.HasTeamRole(TeamRole.Approver))
+        {
+            <div class="alert alert-warning" role="alert">
+                There are @Model.PendingApprovalCount @(Model.PendingApprovalCount > 1 ? "transactions" : "transaction") pending approval, <a asp-controller="Transactions" asp-action="NeedApproval">click here</a> <span> to view them</span>
```

Comment:
This will give you `There are 0 transaction...` and `There are 1 transaction...`.  Probably easiest to leave out the `There are` and change the conditional to `Model.PendingApprovalCount == 1 ? "transaction" : "transactions"`

Reply:
Oh, just noticed 0 won't happen.

Reply:
So maybe change the conditional strings to `is 1 transaction` and `are {n} transactions`?

Reply:
@sprucely Good point. But I'll fix in a later pr.